concordia/static/js/contribute.js

```diff
@@ -429,13 +431,33 @@ var hideTooltipCallback = function() {
     };
     setTimeout(hideTooltip($(this)), 3000);
 };
+
+function trackShareInteraction(element, interactionType) {
+    // Adobe analytics user interaction tracking
+    if (window['loc_ux_tracking'] != undefined) {
```

Comment:
This would more commonly be:

```suggestion
    if ('loc_ux_tracking' in window) {
```